src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring.mdx

```diff
@@ -73,7 +73,7 @@ Because we work at New Relic, we also use New Relic One to analyze and troublesh
     id="alert"
     title="1. Alert notification shows new version crashes"
   >
-    The night after a new release, our on-call developer received an [alert notification on his mobile phone](/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/alerting-new-relic-mobile-apps). The [**Alerts** page](/docs/mobile-monitoring/new-relic-mobile/getting-started/alert-information-new-relic-mobile) indicated that some customers were seeing crashes on the new version of our Android app.
+    The night after a new release, our on-call developer received an [alert notification on his mobile phone](/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/alerting-new-relic-mobile-apps). The [**alerts** page](/docs/mobile-monitoring/new-relic-mobile/getting-started/alert-information-new-relic-mobile) indicated that some customers were seeing crashes on the new version of our Android app.
```

Comment:
@akristen, this reference might need to be capitalized since it is a UI element. Is this the actual name of a page or a shorthand name? You might check with @paperclypse.

Reply:
Sorry, I just noticed this. I think it should be lowercase here.